LOG.tbasicu

```diff
@@ -142,6 +143,10 @@ Type tLogThis
     me.statusbarlPart = statusbarlPart
   
   end function
+  
+  function private_isFlagSet(flagStorage as long, flag as long) as long
```

Comment:
Helpful function to make the code more readable.

I consider it really cool that you use binary flags.

This function will make the code more self documenting, inside it uses logarithm base 2 to do the trick reliably.

Reply:
super!